alica_engine/include/engine/modelmanagement/ModelManager.h

```diff
@@ -51,7 +51,7 @@ class ModelManager
     std::list<std::string> filesParsed;
 
     PlanRepository& _planRepository;
-    std::map<int64_t, AlicaElement*> elements;
+    std::map<int64_t, std::unique_ptr<AlicaElement>> elements;
```

Comment:
better to declare `elements` before `_planRepository` to ensure proper destruction sequence although its most harmless in this case